public/content/it/welcome/welcome-to-d.md

```diff
@@ -0,0 +1,58 @@
+# Benvenuto al tour di D
+
+Benvenuto al tour interattivo del *Linguaggio di Programmazione D*.
```

Comment:
Concordo con l'analisi. Io propenderei per il "nel" perché è più immersivo. Cioè entri nel tour in questo caso, ti addentri. "Al tour" mi sembra qualcosa del tipo "sei arrivato all'inizio del tour" (di certo nessuno direbbe "nell'inizio del tour"). Magari direi "alla prima fase del tour" / "alla seconda" piuttosto che "nella prima" / "nella seconda". Ma sono leggere sfumature di significato indubbiamente :)

Forse un esempio che aiuta a capire cosa intendo è: "bevenuto al bosco incantato" / "benvenuto nel bosco intantato". Nel secondo ci sei immerso dentro. Mi sembra più coinvolgente essere dentro il tour.